presto-native-execution/presto_cpp/main/QueryContextManager.cpp

```diff
@@ -31,6 +31,13 @@ static std::shared_ptr<folly::CPUThreadPoolExecutor>& executor() {
   return executor;
 }
 
+std::shared_ptr<folly::CPUThreadPoolExecutor>& httpProcessingExecutor() {
+  static auto executor = std::make_shared<folly::CPUThreadPoolExecutor>(
```

Comment:
Why not let PrestoServer to own this executor? Rename it to httpServerExecutor? Thanks!

Reply:
Putting all executors in a single place for better maintenance. httpServerExecutor sounds like httpExecSrv which is the executor for iopool.
I'm thinking moving these pools into another file in another PR for isolation. They don't seem to belong to QueryContextManager

Reply:
I think Presto server currently owns these executors. Let's maintain there why we need a separate file?